Source/Eto/Drawing/PaddingConverter.cs

```diff
@@ -12,6 +12,11 @@ namespace Eto.Drawing
 	public class PaddingConverter : TypeConverter
 	{
 		/// <summary>
+		/// The character to split up the string which will be converted
+		/// </summary>
+		private readonly string[] StringSplitter = new string[1] { "," };
```

Comment:
I totally agree about readability and productivity.  The less superfluous code, the better.  

Typically I only sacrifice readability for performance when the impact can be measured.  In this case, these converters can be used thousands of times (depending on your form), so performance is definitely more important and it's nice to see @Panzerfather have actual measurements.

That being said, it would actually be a ton better to have the xaml processing done during compile time so that it has no impact on performance, but that is a problem for another day. (;